lib/library.rb

```diff
@@ -0,0 +1,48 @@
+require 'yaml'
+require 'date'
+
+class Library
+  attr_accessor :bookrack
+  ALLOWED_LOAN_DAYS = 30
+
+
+  def initialize
+    @bookrack = YAML.load_file('./lib/booksdb.yml')
+  end
+
+  def available_books
+    @bookrack.select{|obj| obj[:available] == true}
+  end
+
+  def view_all_books
+    @bookrack
+  end
```

Comment:
This method doesn't help you. Why wouldn't you just use `@bookrack` when you need it?